### PR TITLE
Fix upm-on-replit

### PR DESCRIPTION
**Pull request by masad-frost**
# What Changed 

update nix channel
update packages (including go)
use go module

# Test plan

On replit 

```
$ mkdir testnpm
$ cd testnpm
$ npm init -y
$ ../cmd/upm/upm add left-pad -l nodejs-npm
```

Works

### CODE REVIEW

**Review by ryantm — APPROVED** [review]
Seems kind of strange that we have to get Go from a channel when we have the nixmodule, but I guess the versions don't exactly match yet.

**Comment by masad-frost** [comment]
yeah, I thought we didn't have a 1.20 module, we do. Fixed

File: .replit [hunk 1]
```diff
@@ -1,10 +1,6 @@
...
-
-[languages.go]
-pattern = "**/*.go"
+modules = ["go-1.20:v2-20230911-b5aa5df"]
 
-[languages.go.languageServer]
-start = "gopls"
\ No newline at end of file
+[nix]
+channel = "stable-23_05" # Only channel with go 1.17
```
Comment (cdmistman) [inline .replit:6]: it's actually 1.20 oops

Comment (masad-frost) [reply .replit:6]: the comment is irrelevant, forgot to remove :facepalm:

Comment (masad-frost) [reply .replit:6]: https://github.com/replit/upm/pull/138

**Pull request merged**